build_templates/mkmf.template.rttov.gfortran

```diff
@@ -68,35 +68,29 @@ LD = gfortran
 # for more information on installing RTTOV.
 
 # MODIFY THE FOLLOWING VARIABLES FOR YOUR SYSTEM:
-# If your NETCDF,HDFDIR, or RTTOV environment variables are not set,
+# If your NETCDF, HDFEOS5, or RTTOV environment variables are not set,
 # uncomment the following line and set value to where lib and include
 # are found for the netcdf files that match this compiler.
 # Since netCDF can be built with HDF5, many systems have the HDF5
 # installation in the same place as netCDF.
-
+#
 # NETCDF = /usr/lib/x86_64-linux-gnu
-# HDFDIR = /usr/include/hdf5/serial
+# HDFEOS5 = /usr/include/hdf5/serial
 # RTTOV = ~/research/satellite/rttov
 
-# You will likely not need to modify below this line
+HDFEOS5 = /Users/thoar/gnu/gnu-9.2.0/
 
-HDFINC = $(HDFDIR)/include
-HDFLIB = $(HDFDIR)/lib
-HDFLIBS = -lhdf5hl_fortran -lhdf5_hl -lhdf5_fortran -lhdf5
-
-RTMOD = $(RTTOV)/mod
-RTINC = $(RTTOV)/include
-RTLIB = $(RTTOV)/lib
```

Comment:
I thought this was a bit overly complicated and made it hard for the Intel environment on Cheyenne to look similar.